### PR TITLE
UI feature dashboard

### PR DESCRIPTION
**Pull request by KilianBoute**
Making a dashboard page that includes information about the locations, and a table of all the users under the organisation with features to edit this list.

### CODE REVIEW

**Comment by KilianBoute** [comment]
Basic CRUD operations have been set up.

File: api/controllers/users.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+// user godoc
+// @Router /api/users/{id} [get]
+// @ID getUser
+// @Tags users
+// @Summary Get user by id
+// @Description Get user by id
+// @Param id path int true "User ID"
+// @Success 200 {object} models.User
+func GetUser(id int) models.User {
+	users := database.GetUsers()
```
Comment (cedricve) [inline api/controllers/users.go]: Can we pass in the id to the database methods `GetUser(id)`. When working with a database we can directly query. Move this code to GetUser method of the  file database.

File: api/routers/http/routes.go [hunk 2]
```diff
@@ -17,6 +21,161 @@ func AddRoutes(r *gin.Engine, authMiddleware *jwt.GinJWTMiddleware) *gin.RouterG
...
+		api.GET("/users", func(c *gin.Context) {
+			// Create a list of random users
+			users := controllers.GetUsers()
+			c.JSON(200, gin.H{
+				"data": users,
+			})
+		})
+
+		api.GET("/users/:id", func(c *gin.Context) {
+			// Get the id parameter from the URL
```
Comment (cedricve) [inline api/routers/http/routes.go]: Ideally we have as little logic in the router handlers, so it stays clear what HTTP methods we do have available. Would expect something like this:


    r.GET("/videowalls/:id", func(c *gin.Context) {
        middleware.WithUserMetadata(c, log, controllers.GetVideowall)
    })

**Review by cedricve — CHANGES_REQUESTED** [review]

**Review by cedricve — APPROVED** [review]
Luuking guud

**Pull request merged**